soccer/gameplay/evaluation/defensive_positioning.py

```diff
@@ -312,7 +312,8 @@ def goalside_mark_segment(
 ## Finds the line segment between the mark_pos and the ball
 # @param mark_pos: Point (usually robot position) to defend against
 # @return: LineSegment to defend on
-def ballside_mark_segment(mark_pos: robocup.Point, ball_pos=None):
+def ballside_mark_segment(mark_pos: robocup.Point,
+                          ball_pos: robocup.Point = None) -> robocup.Line:
```

Comment:
Does this need to be an optional to have the none type?

Reply:
yeah, woops